NoteworthyUserManual.md

```diff
@@ -0,0 +1,46 @@
+# Description 
+Welcome to Noteworthy! We are the #1 Notetaking Application For Markdown files. Some of our current available functions >>> include writing, viewing, searching, listing, deleting, clearing, and converting to PDF. If you have any questions relating to Markdown the link below provides useful information
```

Comment:
\# Notetaking Application For Markdown files doesn't need to be caps cause it isn't a title